share/mdnjs/fetch.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 python fetch.py --cachedir downloads \
-                --cachejournal ".cachejournal" \
+                --cachejournal ".cachejournal_api" \
+                --patt "en-US/docs/Web/API" \
+                --sitemap http://developer.mozilla.org/sitemaps/en-US/sitemap.xml \
+                --sleep 5
+
+python fetch.py --cachedir downloads \
+                --cachejournal ".cachejournal_js" \
                 --patt "en-US/docs/Web/JavaScript/Reference/Global_Objects" \
                 --sitemap http://developer.mozilla.org/sitemaps/en-US/sitemap.xml \
                 --sleep 5
+
+cat .cachejournal_api >> .cachejournal
+cat .cachejournal_js >> .cachejournal
```

Comment:
Thanks @jdorweiler, I understand now., I've somehow missed the `--patt` part. Feel free to forget about my previous comment =)